docs/source/quickstart_mlops.rst

```diff
@@ -0,0 +1,138 @@
+Choose **Chart view**. Choose the **Parallel coordinates** graph and configure it to show the **lr** and **momentum** coordinates and the **test_rmse** metric. Each line in this graph represents a run and associates the parameters and resulting error. 
+
+.. image:: _static/images/quickstart_mlops/mlflow_ui_chart_view.png
+    :width: 800px
+    :align: center
+    :alt: Screenshot of MLflow tracking UI parallel coordinates graph showing runs
+
+The red graphs on this graph are runs that fared poorly. The lowest one has set both **lr** and **momentum** to 0.0 and has an RMSE of ~0.89. The other red lines show that high **momentum** can also lead to poor results with this architecture. 
+
+The graphs shading towards blue are runs that fared better. Hover your mouse over individual runs to see their details.
+
+Register your best model
+------------------------
+
+Choose the best run and register it as a model. In the **Table view**, choose the best run. In the **Run Detail** page, open the **Artifacts** section and select the **Register Model** button. In the **Register Model** dialog, enter a name for the model, such as ``wine-quality``, and click **Register**.
+
+Now, your model is available for deployment. You can see it in the **Models** page of the MLflow UI. Open the page for the model you just registered.
+
+You can add a description for the model, add tags, and easily navigate back to the source run that generated this model. You can also transition the model to different stages. For example, you can transition the model to **Staging** to indicate that it is ready for testing. You can transition it to **Production** to indicate that it is ready for deployment.
+
+Transition the model to **Staging** by choosing the **Stage** dropdown:
+
+.. image:: _static/images/quickstart_mlops/mlflow_registry_transitions.png
+    :width: 800px
+    :align: center
+    :alt: Screenshot of MLflow tracking UI models page showing the registered model
+
+Serve the model locally
+----------------------------
+
+MLflow allows you to easily serve models produced by any run or model version. You can serve the model you just registered by running:
+
+.. code-block:: bash
+
+  mlflow models serve -m "models:/wine-quality/Staging" --port 5002
+
+(Note that specifying the port as above may be necessary if you are running the tracking server on the same machine at the default port of **5000**.)
+
+You could also have used a `runs:/<run_id>` URI to serve a model, or any supported URI described in :ref:artifact_stores:.
+
+To test the model, you can send a request to the REST API using the ``curl`` command:
+
+.. code-block:: bash
+
+  curl -d '{"dataframe_split": {"columns": ["fixed acidity","volatile acidity","citric acid","residual sugar","chlorides","free sulfur dioxide","total sulfur dioxide","density","pH","sulphates","alcohol"], "data": [[7,0.27,0.36,20.7,0.045,45,170,1.001,3,0.45,8.8]]}}' -H 'Content-Type: application/json' -X POST localhost:5002/invocations
+
```

Comment:
Can a small mention be made here about the `signature` that was defined within the example during the logging Callback's exit and how the input JSON `data` payload must match the input signature schema? 
The concept of signatures has historically been purely optional and only really used for model serving, but it is increasingly going to be a mandatory element to log. 
It will be important to mention this here as a friendly reminder to users that we're doing schema validation and that if the field names and data types don't match, we'll throw.

Reply:
Done in 61b31041